### PR TITLE
Use <md-icon> instead of <i class="material-icons"> in demos.

### PR DESCRIPTION
**Pull request by dozingcat**
Also removed vertical-align in mdIcon styles because it causes the button icons to be misaligned.

### CODE REVIEW

**Comment by jelbourn** [comment]
LGTM, just needs rebase

**Comment by dozingcat** [comment]
Done. @kara might want to double check the merges.

File: src/components/button/_button-base.scss [hunk 1]
```diff
@@ -107,7 +107,7 @@ $md-mini-fab-padding: 8px !default;
   height: $size;
   padding: 0;
 
-  i {
+  md-icon {
```
Comment (kara) [inline src/components/button/_button-base.scss]: This should actually be `i`, not `md-icon`.  `md-icon` is already aligned correctly; it's just regular `i` that need the padding.

Comment (dozingcat) [inline src/components/button/_button-base.scss]: Done

**Comment by kara** [comment]
LGTM

**Pull request merged**